app/models/SicAndCompliance.scala

```diff
@@ -26,7 +26,8 @@ case class SicAndCompliance(description: Option[BusinessActivityDescription] = N
                             supplyWorkers: Option[SupplyWorkers] = None,
                             workers: Option[Workers] = None,
                             intermediarySupply: Option[IntermediarySupply] = None,
-                            hasLandAndProperty: Option[Boolean] = None)
+                            hasLandAndProperty: Option[Boolean] = None,
+                            otherBusinessInvolvement: Option[Boolean] = None)
```

Comment:
The model would also need a refactor to its reads and writes to support the new field